### PR TITLE
LibWeb: rename 'cross-origin opener policy' to 'opener policy'

### LINKED ISSUE #1387: Rename "cross-origin opener policy" to "opener policy"
As changed here: https://github.com/whatwg/html/pull/10579

As well as renaming our [`CrossOriginOpenerPolicy`](https://github.com/LadybirdBrowser/ladybird/blob/master/Userland/Libraries/LibWeb/HTML/CrossOrigin/CrossOriginOpenerPolicy.h), we'll need to update any spec comments that refer to it. That'll involve a lot of looking through the [changes](https://github.com/whatwg/html/pull/10579/files) in that PR, finding where we implement that part of the spec, and changing the text.

### PR DESCRIPTION
**Pull request by 0xkon1**
Closes #1387

### CODE REVIEW

**Review by AtkinsSJ — CHANGES_REQUESTED** [review]
Thanks for doing this!

I've marked a few places where your changes don't quite match the spec text. We're quite fussy about that. :^)

Also in two places the spec still says "cross-origin opener policy". That seems to be a mistake in the spec so while I've noted them, you can ignore them. I've opened https://github.com/whatwg/html/pull/10631 to fix the spec.

When making corrections, please make the changes to your existing commit. [This video](https://www.youtube.com/watch?v=ElRzTuYln0M) is good at explaining how to do that in case you're not super familiar with git. You can also ask here or on Discord if you get confused.

File: Userland/Libraries/LibWeb/HTML/Navigable.cpp [hunk 1]
```diff
@@ -1607,17 +1607,17 @@ WebIDL::ExceptionOr<JS::GCPtr<DOM::Document>> Navigable::evaluate_javascript_url
     // FIXME: 13. Let finalSandboxFlags be policyContainer's CSP list's CSP-derived sandboxing flags.
     auto final_sandbox_flags = SandboxingFlagSet {};
 
-    // 14. Let coop be targetNavigable's active document's cross-origin opener policy.
-    auto const& coop = active_document()->cross_origin_opener_policy();
+    // 14. Let coop be targetNavigable's active document's opener policy.
```
[the remark this answers is not in the document]
Comment (AtkinsSJ) [reply Userland/Libraries/LibWeb/HTML/Navigable.cpp:1610]: This has been corrected now.

**Comment by 0xkon1** [comment]
Thank you for the feedback. I could have done it easier by just copying the spec text. Should al be corrected now.

**Review by AtkinsSJ — APPROVED** [review]
Looks good! Thanks again. :^)

**Comment by 0xkon1** [comment]
Happy to be able to contribute! Thanks for the help.
\

**Pull request merged**